portfolio/src/main/webapp/script.js

```diff
@@ -15,18 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function getData() {
+  fetch('/data').then(response => response.json()).then((data) => {
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    const dataElement = document.getElementById('data-container');
+    dataElement.innerHTML = '';
+    dataElement.appendChild(createListElement(data[0]));
+    dataElement.appendChild(createListElement(data[1]));
+    dataElement.appendChild(createListElement(data[2]));
```

Comment:
If you want you can add all of the elements in data by doing:
```suggestion
    data.map(creatListElement)
            .forEach(dataElement.appendChild);
```